charts/flyte-core/templates/propeller/crds/flyteworkflow.yaml

```diff
@@ -21,4 +26,7 @@ spec:
           type: object
           x-kubernetes-preserve-unknown-fields: true
           properties:
+{{- else }}
+  version: v1alpha1
```

Comment:
should this be plural?  or the one above singular?

Reply:
This is the deprecated form.. you could only have one... the new api allows multiple versions declared in the CRD...